src/chains/ethereum/ethereum/tests/contracts/DebugStorage.sol

```diff
@@ -5,20 +5,27 @@ contract DebugStorage {
   uint public value;
   uint public valueTwo;
   string public valueThree;
+  uint public value4;
```

Comment:
is there a reason that this is `value4` and not `valueFour` like the others?

Reply:
no reason. fixed in https://github.com/trufflesuite/ganache/pull/3470/commits/d2da0f52f5562526235dc7b005da76dfb7dd79e7